Makefile

```
@@ -67,7 +67,8 @@ pre-requirements: ## install Python requirements for running pip-tools
 	pip install -qr requirements/edx/pip-tools.txt
 
 requirements: pre-requirements ## install development environment requirements
-	pip-sync -q requirements/edx/development.txt requirements/edx/private.*
+	# Wildcard "txt*" is so private.txt is included only if it exists
+	pip-sync -q requirements/edx/development.txt requirements/edx/private.txt*
```

This doesn't seem to be doing what it says it should.
I get the following error running `make requirements` with this change (inside devstack):

> Error: Invalid value for '[SRC_FILES]...': Path 'requirements/edx/private.txt*' does not exist.


You may need to rely on `make`'s wildcard support, instead of shell expansion:

```suggestion
	pip-sync -q requirements/edx/development.txt $(wildcard ./requirements/edx/private.txt)
```

---

Tested this works in my devstack ^